.github/workflows/coverage.yml

```diff
@@ -63,10 +63,17 @@ jobs:
     - name: Install binutils and rustfilt
       run: cargo install cargo-binutils rustfilt
 
+    - name: Start private tangle
+      uses: './.github/actions/private-tangle/setup'
+
     - name: Run tests and collect coverage
       run: bash ./coverage.sh
```

Comment:
Also need to add `-- --include-ignored` in coverage.sh if you want to test the ignored tests

Reply:
Thanks